data-otservbr-global/scripts/lib/register_actions.lua

```diff
@@ -651,6 +651,11 @@ function onUsePick(player, item, fromPosition, target, toPosition, isHotkey)
 		--The Ice Islands Quest, Nibelor 1: Breaking the Ice
 		local missionProgress = player:getStorageValue(Storage.Quest.U8_0.TheIceIslands.Mission02)
 		local pickAmount = player:getStorageValue(Storage.Quest.U8_0.TheIceIslands.PickAmount)
+
+		if pickAmount < 0 or pickAmount == nil then
+			player:setStorageValue(Storage.Quest.U8_0.TheIceIslands.PickAmount, 0)
+		end
+
```

Comment:
i don't think we need this check

Reply:
Yep, removed this and retested, still works, imma remove this block of code.

Reply:
I didn't fully understand the math.max calculation for the pickAmount hence I added this xD